### PR TITLE
Update estimate when using a steady ticker

### LINKED ISSUE #493: Custom keys on ProgressStyle do not work together with `enable_steady_tick`
It seems as though there are the following two interactions in indicatif 0.17.2, which were not present in 0.17.0-rc4:

- If `set_style` is used _before_ `enable_steady_tick`, the steady tick takes effect, but any custom keys that are set on the ProgressStyle using its `with_key` method are never updated.
- If `set_style` is used _after_ `enable_steady_tick`, then the custom keys work but the steady tick is disabled.

### PR DESCRIPTION
**Pull request by aawsome**
closes #493

### CODE REVIEW

File: src/progress_bar.rs [hunk 1]
```diff
@@ -641,7 +641,7 @@ impl TickerControl {
             }
 
             state.state.tick = state.state.tick.saturating_add(1);
-            state.draw(false, Instant::now()).ok();
+            state.update_estimate_and_draw(Instant::now());
```
Comment (aawsome) [inline src/progress_bar.rs:644]: Actually, we could simply call `state.tick(Instant::now())` here.

Comment (aawsome) [reply src/progress_bar.rs]: I changed this.

**Comment by tobloef** [comment]
Awesome 🎉  Could we get this merged?

**Comment by djc** [comment]
@chris-laplante do you want to review this?

**Review by chris-laplante — APPROVED** [review]
LGTM!

**Pull request merged**